com.unity.sg2/Editor/GraphDeltaRegistry/Registry/NodeUIInfo.cs

```diff
@@ -21,29 +22,38 @@ internal class StaticNodeUIDescriptorBuilder : INodeUIDescriptorBuilder
     /// </summary>
     internal class NodeUIInfo
     {
-        readonly Dictionary<RegistryKey, INodeUIDescriptorBuilder> builders = new ();
+        readonly Dictionary<RegistryKey, INodeUIDescriptorBuilder> factories = new ();
 
         public void Register(RegistryKey key, NodeUIDescriptor descriptor)
             => Register(key, new StaticNodeUIDescriptorBuilder(descriptor));
 
         public void Register(RegistryKey key, INodeUIDescriptorBuilder descriptor)
-            => builders[key] = descriptor;
+            => factories[key] = descriptor;
 
         public NodeUIDescriptor GetNodeUIDescriptor(RegistryKey key, NodeHandler nodeInstance)
-            => builders.ContainsKey(key) ? builders[key].CreateDescriptor(nodeInstance) : CreateDefaultDescriptor();
+            => factories.ContainsKey(key) ? factories[key].CreateDescriptor(nodeInstance) : CreateDefaultDescriptor(key);
```

Comment:
Maybe on register we should register with a DefaultUIDescriptorBuilder (similar to the StaticUIDescriptorBuilder) that just uses the localIDs for nodes/ports for all the details.